File: RTRefSection.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace LORD2
{
    public class RTRefSection
    {
        public Dictionary<string, int> Labels;
        public string Name;
        public List<string> Script;

        public RTRefSection(string name)
        {
            Labels = new Dictionary<string, int>(StringComparer.OrdinalIgnoreCase);
            Name = name;
            Script = new List<string>();
        }
    }
}

File: rtrefsection.pas
unit RTRefSection;

interface

uses
  contnrs, Classes;

type
  TRTRefSection = class
  public
    Labels: TFPHashObjectList;
    Name: String;
    Script: TStringList;

    constructor Create(ASectionName: String);
    destructor Destroy;
  end;

implementation

constructor TRTRefSection.Create(ASectionName: String);
begin
  inherited Create;
  Labels := TFPHashObjectList.Create;
  Name := ASectionName;
  Script := TStringList.Create;
end;

destructor TRTRefSection.Destroy;
begin
  inherited Destroy;
end;

end.

